ares/n64/cpu/cpu.cpp

```diff
@@ -147,7 +147,9 @@ auto CPU::instructionEpilogue() -> s32 {
     icache.step(ipu.pc, devirtualizeFast(ipu.pc));
   }
 
-  ipu.r[0].u64 = 0;
```

Comment:
Consider `assert(ipu.r[0].u64 == 0);` in the recompiler code path. This is a careful dance it's good to be on the safe side in debug builds.

Reply:
I'll add that